tests/qchem/openfermion_pyscf_tests/conftest.py

```diff
@@ -16,10 +16,25 @@
 """
 import shutil
 
+import numpy as np
 import pytest
+from packaging.version import Version
```

Comment:
Any reason we use `packaging` instead of `importlib` for this?

Reply:
No I don't have personal preference here. I used this only because I saw we used it other places in PL before.

Reply:
If packaging is already in the venv as a dep, that's fine. importlib would be relying completely on the Python standard lib though. If this paradigm is used elsewhere, we can keep it for now.